File: flutter-exercises/places_app/lib/models/place.dart
import 'dart:io';

import 'package:uuid/uuid.dart';

const uuid = Uuid();


class PlaceLocation {
  const PlaceLocation({
    required this.lat,
    required this.long,
    required this.alt,
  });

  final double lat;
  final double long;
  final double alt;
}

class Place {
  Place({
    required this.title,
    required this.image,
    required this.location,
  }) : id = uuid.v4();

  final String id;
  final String title;
  final File image;
  final PlaceLocation location;
}

File: reactnative-exercises/places_app/models/place.js
import { v4 as uuidv4 } from 'uuid';

export class Place {
    constructor(title, imageUrl, location) {
        this.title = title;
        this.imageUrl = imageUrl;
        this.location = { lat: location.lat, long: location.long };
        // this.id = uuidv4().toString();
        this.id = new Date().toString() + Math.random().toString();
    }
}